pydarn/plotting/maps.py

```diff
@@ -70,7 +70,7 @@ def plot_mapdata(cls, dmap_data: List[dict], ax=None,
                      record: int = 0, start_time: dt.datetime = None,
                      time_delta: float = 1,  alpha: float = 1.0,
                      len_factor: float = 150, color_vectors: bool = True,
-                     cmap: str = None, colorbar: bool = True,
+                     cmap: str = None, colorbar: bool = True, contour_colorbar=False,
```

Comment:
```suggestion
                     cmap: str = None, colorbar: bool = True, contour_colorbar: bool = False,
```

Reply:
Line too long too, bump to next line and tidy up

Reply:
Also think this should be default true, if contours aren't filled it won't plot anyway.
```suggestion
                     cmap: str = None, colorbar: bool = True, contour_colorbar: bool = True,
```